openarm_bringup/launch/openarm.launch.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2024, Stogl Robotics Consulting UG (haftungsbeschränkt)
+# Copyright 2024 Stogl Robotics Consulting UG (haftungsbeschränkt)
```

Comment:
Have we changed this file?
If so, we need our copyright line.

Reply:
fix: [f67a461](https://github.com/reazon-research/openarm_ros2/pull/13/commits/f67a461475d7b5d9ec99db4af9dbffd926a79251) I'm sorry. I thought there was no changed but it does. So, I added the license headers too.